app/assets/stylesheets/common/rich-editor/composer-image-gallery.scss

```
@@ -0,0 +1,61 @@
+@use "lib/viewport";
+
+.composer-image-gallery {
+  &__mode-buttons {
+    position: absolute;
+    right: var(--space-2);
+    top: var(--space-2);
+    display: flex;
```

```suggestion
    display: flex;
    z-index: 1;
```

I can tell we don't show images below these buttons, but to be safe, adding a z-index here would ensure they would always be above the other content on this block.